compliant-standards.md

```diff
@@ -0,0 +1,150 @@
+# EMC Compliant Standards
+
+## 1. Introduction
+
+### 1.1 What is this document?
+
+The Email Markup Consortium (EMC) is working to improve email markup, the code used to structure and style content within an email. This document lays out the standards of what that markup should look like.  
+  
+This document is aimed at top level ideas of what the email markup should do, not specific ways to code.
+
+### 1.2 Who is this document for?
+
+This document is for anyone creating email markup, processing email markup, or displaying email markup. This includes (but isn’t limited to) email developers, email creation tools, email sending tools, and email clients.
+
+For this to work it will require developers to make improvements to the markup at the point of creation. As long as that markup is valid it should remain in a recognisable state with minimal changes through to the point the recipient opens the email. This means any processing of that markup by an ESP, mail server, email client or any other technology that touches it, should also follow the guidelines to preserve the standard.
+
+### 1.3 Why is this needed?
+
+The current state of email markup varies a lot. Email markup goes through several stages from development to being read by the recipient. Each of these stages can make changes to the markup. Depending on what services are used this can result in very different code being seen by the user as each service may take a different approach. This document sets out some top level goals for how the markup should be treated.  
+  
+Without this, there can be a stalemate in seeing progress in email markup. Senders won’t include updates that email clients don’t support and emails clients don’t see the need to include updates that senders aren’t including. We need changes on both sides.
+
+### 1.4 What this document is not
+
+This document is not a “how to” guide on how you can be compliant with these standards.
+
+This document does not cover email design or content choices.
+
+This document does not cover email sending protocols.
+
+This document does not take into consideration current levels of markup support; these are targets to strive for.
+
+## 2. Accessibility
+
+Accessibility is key to achieving our markup standards. When email markup is properly designed and coded, people of all abilities can use it.
+
+### 2.1 Content should fit the container
+
+Email content should respond to fit the size of the container it is loaded in. A recipient should not have to scroll on more than one axis to see parts of the email, nor should content be cut off.
+
+> Content should adjust to fit the available space with fluid content or @media queries.
```

Comment:
I don't think block quotes is the most suitable way to format this. The same applies to the other points. 

If it is an example, perhaps having "Examples" as a subheading would make things clearer. If it is further explanation, nothing (including the block quote) is required.

Reply:
Yeah it's a tricky one, i was thinking of a way to seperate it visually but it's hard in markdown, I think a subheading is a good idea.  Also if I'm planning on only doing it visually it's not accessible